### DISCUSSION TITLE
Any way (environmental variable? config?) to change the port back to 80?

### DISCUSSION THREAD

**Discussion by markuskonojacki** [Lissy93/dashy#1553] [Help and Support] [answered]
Hey there, love dashy. :)
Is there any intended way to change the port back? My usecase is, that I'm binding containers I use often to their own ip.
I like to call dashy on 10.10.10.10 as my homelab index without the need to now add :8080.
I could access it through a reverse proxy, but that's not something I'd like to do currently.

**Comment by Automayted** [comment, marked as answer]
@markuskonojacki, I use macvlan networking and prefer Dashy to run on port 80 as well. Adding this to my compose file worked for me:

```
environment:
  - NODE_ENV=production
  - PORT=80
```
